pyproject.toml

```diff
@@ -1,23 +1,24 @@
 [tool.poetry]
-name = "data-product-definition-tooling"
-version = "0.0.13"
-description = "Data Product Definition Tooling"
+name = "ioxio-data-product-definition-tooling"
+version = "0.0.1"
+description = "IOXIO Data Product Definition Tooling"
 authors = ["IOXIO Ltd"]
 license = "BSD-3-Clause"
 readme = "README.md"
-repository = "https://github.com/ioxio-dataspace/data-product-definition-tooling"
+repository = "https://github.com/ioxio-dataspace/ioxio-data-product-definition-tooling"
 packages = [
-    {include="converter", from="."}
+    {include="definition_tooling", from="."},
```

Comment:
I'm a bit considering that could we do some sort of tricks here in order to not have to change the definitions to include from `definition_tooling.converter` and thus have to update the guides etc. But I believe if we do that, then we'll get problems with the shared stuff that is in the `definition_tooling` directory. So likely not something we can do, but maybe pause for a moment and consider if this or something else could worl.

Reply:
Yeah, makes sense. Added `converter` to be exportable too. Now it's backward compatible